drop-storage/src/lib.rs

```diff
@@ -177,6 +225,12 @@ impl Storage {
     pub async fn insert_transfer_failed_state(&self, transfer_id: Uuid, error: u32) -> Result<()> {
         let tid = transfer_id.hyphenated();
 
+        info!(
+            self.logger,
+            "Inserting transfer failed state";
+            "transfer_id" => tid.to_string(),
```

Comment:
Do we actually implement structured logging with the log callback? I guess not

Reply:
That was a  good catch, my eyes missed that one. It is now implemented in our drain